hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/snapshot/TestExportSnapshot.java

```diff
@@ -281,6 +283,39 @@ public void testExportWithResetTtl() throws Exception {
     }
   }
 
+  @Test
```

Comment:
By default, this test case does not execute  beacuse of  https://github.com/apache/hbase/blob/master/hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/snapshot/TestExportSnapshot.java#L78

After manually modifying it, the test case passes.